packages/cisco/data_stream/ios/elasticsearch/ingest_pipeline/default.yml

```
@@ -47,7 +48,19 @@ processors:
       field: message
       pattern: "list %{cisco.ios.access_list} %{_temp_.event.outcome} %{network.iana_number} %{source.address} %{} %{destination.address}, %{source.packets} packet"
       if: "['IPACCESSLOGNP', 'ACCESSLOGNP'].contains(ctx?.event?.code)"
-
+  - dissect:
```

```suggestion
  - set:
       field: event.action
       value: "multicast-join"
       if: ctx?.event?.code == "INVALID_RP_JOIN"
  - set:
       field: event.outcome
       value: "failure"
       if: ctx?.event?.code == "INVALID_RP_JOIN"
  - set:
       field: event.reason
       value: "Invalid RP"
       if: ctx?.event?.code == "INVALID_RP_JOIN"
  - dissect:
```

Following suggestion from https://github.com/elastic/integrations/pull/896/files#r610127848

---

done